src/app/models/platform-api/requests/tokens/tokens-filter.ts

```diff
@@ -8,6 +8,20 @@ export interface ITokensRequest {
   keyword?: string;
 }
 
+export enum TokenOrderByTypes {
+  Name = 'Name',
+  Symbol = 'Symbol',
+  Default = 'Default',
+  PriceUsd = 'PriceUsd',
+  DailyPriceChangePercent = 'DailyPriceChangePercent'
+}
+
+export enum TokenProvisionalTypes {
+  All = 'All',
+  Provisional = 'Provisional',
+  NonProvisional = 'NonProvisional'
+}
```

Comment:
Perfect, can you just set the types of `TokensFilter.provisional` and `TokensFilter.orderBy` below to use the Enum type instead of string.

Reply:
Done